src/components/Node.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <li>
+    <template v-if="hasChildren">
+      <span class="open-toggle" @click.prevent="toggle">
+        <Icon :name="icon" class="fa-xs" fixed-width />
+      </span>
+    </template>
+
+    <span class="node version" v-if="routable">
+      <router-link
+        :to="{ path: 'reader', query: { urn: metadata.firstPassageUrn } }"
+      >
+        {{ metadata.workTitle }}
+      </router-link>
+    </span>
+    <span v-else class="node monospace">
+      <tt>{{ urn }}</tt>
+    </span>
+
+    <ul class="node-tree" v-if="expanded">
+      <Node
+        v-for="(node, index) in node.children"
+        :key="index"
+        :node="node"
+      ></Node>
+    </ul>
+  </li>
+</template>
+
+<script>
+  import Icon from '@/components/Icon.vue';
+
+  export default {
+    name: 'Node',
+    props: ['node'],
+    components: {
+      Icon,
+    },
+    data() {
+      return {
+        expanded: false,
+      };
+    },
+    computed: {
+      urn() {
+        // TODO: Cast to URN post-refactor.
+        return this.node.data.urn;
+      },
+      kind() {
+        return this.node.data.kind;
+      },
+      metadata() {
+        return this.node.data.metadata;
+      },
+      routable() {
+        return this.kind === 'version';
+      },
+      icon() {
```

Comment:
again, not a blocker here, but working through our larger localization story (informed by Digital Sirah), I think `scaife-widgets` can re-use some of our RTL widgets to handle things like chevrons

Reply:
Good plan, I'll look at that.